presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java

```diff
@@ -35,16 +35,19 @@ public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
             throws SQLException
     {
         super(connectorId, config, "\"", new SQLServerDriver());
+        connectionProperties.setProperty("initializationString", "SET QUOTED_IDENTIFIER, ANSI_NULLS, CONCAT_NULL_YIELDS_NULL, ARITHABORT ON;");
+        connectionProperties.setProperty("sendStringParametersAsUnicode", "false");
+        connectionProperties.setProperty("applicationName", "Presto");
     }
 
     @Override
     public void commitCreateTable(JdbcOutputTableHandle handle)
     {
         StringBuilder sql = new StringBuilder()
                 .append("sp_rename ")
-                .append(singleQuote(handle.getCatalogName(), handle.getSchemaName(), handle.getTemporaryTableName()))
+                .append("'" + quoted(handle.getCatalogName(), handle.getSchemaName(), handle.getTemporaryTableName()) + "'")
                 .append(", ")
-                .append(singleQuote(handle.getTableName()));
+                .append("'" + handle.getTableName() + "'");
```

Comment:
Didn't run it, but above you had `quoted` here you don't have. 

The convenience function `singleQuote` looked useful in this case.